flaky-tests/generating-junit-reports.md

```diff
@@ -6,13 +6,13 @@ description: How to generate JUnit Reports for common test runners
 
 ## Jest
 
-1. Install [`jest-junit`](https://github.com/jest-community/jest-junit)
+1. Install [`jest-junit`](https://github.com/jest-community/jest-junit).
```

Comment:
We need to also update this to mention how to get the `file` field populated. Jest requires the `JEST_JUNIT_ADD_FILE_ATTRIBUTE` to be set.